frontstage/templates/surveys/surveys-add.html

```diff
@@ -0,0 +1,57 @@
+{% import 'partials/section.html' as section %}
```

Comment:
It would be good to go through these html files and make the indenting consistent, the browser wont care but they're be easier to manage if it's more consistent.